.ci/setup_env_sles.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -e
+
+cidir=$(dirname "$0")
+source "/etc/os-release" || source "/usr/lib/os-release"
+source "${cidir}/lib.sh"
+
+echo "Add repo for perl-IPC-Run"
+perl_repo="https://download.opensuse.org/repositories/devel:languages:perl/SLE_12_SP3/devel:languages:perl.repo"
+sudo -E zypper addrepo ${perl_repo}
+sudo -E zypper refresh
+
+echo "Install perl-IPC-Run"
+sudo -E zypper -n install perl-IPC-Run
+
+echo "Add repo for moreutils"
+moreutils_repo="https://download.opensuse.org/repositories/utilities/SLE_12_SP3_Backports/utilities.repo"
+sudo -E zypper addrepo ${moreutils_repo}
+sudo -E zypper refresh
+
+echo "Install chronic"
```

Comment:
just a thought - if we can, we could move chronic to be the first thing to install, and then we can use it for all the other item installs (like the perl stuff above) - presuming chronic doesn't need perl that is?

Reply:
```
$ file $(which chronic)                                                                                                                            
/usr/bin/chronic: Perl script text executable
```